src/store/mutations.js

```diff
@@ -284,13 +284,13 @@ export default {
 			return
 		}
 
-		const mailbox = state.mailboxes[envelopes[0].mailboxId]
```

Comment:
assumes that all envelopes passed have the same mailboxId. That is not true for unified mailboxes because their envelopes still point to the mailboxId of their original mailbox.